test/unit/services/model/RejectionReasonSpec.scala

```diff
@@ -243,6 +246,71 @@ class RejectionReasonSpec extends UnitSpec with ExportsTestData with Injector {
       RejectionReason.url(url, declaration, Some(pointer)) mustBe expectedUrl
     }
 
+    "correctly build a pointer [declaration.declarantDetails.details.eori] base url for a CLEARANCE Declaration" in {
+
+      val sections = Seq(
+        PointerSection("declaration", PointerSectionType.FIELD),
+        PointerSection("declarantDetails", PointerSectionType.FIELD),
+        PointerSection("details", PointerSectionType.FIELD),
+        PointerSection("eori", PointerSectionType.FIELD)
+      )
+      val pointer = Pointer(sections)
+      val url = "/customs-declare-exports/declaration/URL_PATH"
+      val declaration = aDeclaration(withType(DeclarationType.CLEARANCE))
+
+      val expectedUrl = s"/customs-declare-exports/declaration/declarant-details"
+
+      RejectionReason.url(url, declaration, Some(pointer)) mustBe expectedUrl
+    }
+
+    "correctly build a pointer [declaration.declarantDetails.details.eori] base url for a CLEARANCE Declaration with EIX" in {
+
+      val sections = Seq(
+        PointerSection("declaration", PointerSectionType.FIELD),
+        PointerSection("declarantDetails", PointerSectionType.FIELD),
+        PointerSection("details", PointerSectionType.FIELD),
+        PointerSection("eori", PointerSectionType.FIELD)
+      )
+      val pointer = Pointer(sections)
+      val url = "/customs-declare-exports/declaration/URL_PATH"
+      val declaration = aDeclaration(withType(DeclarationType.CLEARANCE), withIsExs(), withPersonPresentingGoodsDetails(Some(Eori("GB201920200000"))))
+
+      val expectedUrl = s"/customs-declare-exports/declaration/person-presenting-goods"
+
+      RejectionReason.url(url, declaration, Some(pointer)) mustBe expectedUrl
+    }
+
+    "correctly build a pointer [declaration.declarantDetails.details.eori] base url for a STANDARD Declaration" in {
```

Comment:
Is it possible to visit this page for other journeys as well? If so, maybe change this test to be run against all of them using `JourneyTypeTestRunner` ?